Doc/tools/templates/customsourcelink.html

```diff
@@ -8,6 +8,12 @@ <h3>{{ _('This Page') }}</h3>
             rel="nofollow">{{ _('Show Source') }}
         </a>
       </li>
+      {% if language != "en" %}
+      <li>
+        <a href="https://github.com/python/python-docs-{{ language }}/blob/{{ version }}/{{ pagename }}.po"
+           rel="nofollow">{{ _('Show translation source') }}</a>
```

Comment:
Or to match the others, even though the source English will never be shown 🙃

```suggestion
           rel="nofollow">{{ _('Show Translation Source') }}</a>
```